### ISSUE TITLE
Initialization of DIDDoc fails (except in rare cases)

### ISSUE THREAD

**Issue by pauldesmondparker** [tuum-tech/identify#74]
# Description

```
Chromium 116.0.5845.96 Arch Linux
branch: master
Metamask Flask 10.34.4-flask.0
```
Cannot get the `Account Info` to populate, fails on entry of `account id`. One (1) works, two (2) does not.
![image](https://github.com/tuum-tech/identify/assets/114397409/8c84e025-752e-4c60-bef9-50f0c9a69577)
![image](https://github.com/tuum-tech/identify/assets/114397409/491d3737-c815-4b77-84ff-8e54c738ca8c)
![image](https://github.com/tuum-tech/identify/assets/114397409/a7fb921e-f6d0-463d-95e8-19ecec496811)

# Remediation
> [!Note]
> It has been intermittently possible to remove flask; 1. add hedera testnet network, 2. import testnet account; and then it would work.
> But that hasn't worked consistently.

# Reproduce
1. Fresh git clone
2. yarn install && yarn start
3. remove and install flask
4. connect :white_check_mark: 
5. send hello message :negative_squared_cross_mark: 

> [!Important]
> `{code: -32603, message: "Error while trying to get the account: Error: Coul…era account info using the accountId '0.0.487338'" ...}`

https://hashscan.io/testnet/account/0.0.487338

**Comment by kpachhai** [comment]
Are you connecting to this hedera account via metamask account or external account? https://docs.tuum.tech/identify/tutorials/hedera-hashgraph

Also what's the identity snap version that's being used? Recently, we released a new version 1.5.0.

**Comment by pauldesmondparker** [comment]
My bad. My mistake.

> Also what's the identity snap version that's being used? Recently, we released a new version 1.5.0.
![image](https://github.com/tuum-tech/identify/assets/114397409/6dbcccc9-d5b2-4035-a2c4-92e22712fcdb)

I'm using an external account, but got the `accountId` from the ED25519 account mixed up with the ECDSA account.

> [!Note]
> I imported my account into metamask with the private key from (4).  Which gave the account (1).
> But then I used `accountId` (2) instead of (3) 

![image](https://github.com/tuum-tech/identify/assets/114397409/3b1864af-555e-4433-a54b-9554c9f90930)
